src/components/FilterPanel/FilterPanelLabel/FilterPanelLabel.js

```diff
@@ -59,6 +60,11 @@ FilterPanelLabel.propTypes = {
    * Optional class name for the count.
    */
   countClassName: PropTypes.string,
+
+  /**
+   * The `title` attribute that is applied to the label node.
+   */
+  title: PropTypes.string,
```

Comment:
I don't know where else add this comment, but the `FilterPanelCheckbox` component also needs to accept the `title` prop. When I updated the storybook for it to test, the title did not get applied:

![image](https://user-images.githubusercontent.com/5973294/82951566-05749700-9f6d-11ea-9160-2fa9a0c26218.png)

![image](https://user-images.githubusercontent.com/5973294/82951628-24732900-9f6d-11ea-9dd0-64e26ec644b2.png)

With this in mind, it might also be good to add a knob for the title prop for this component.